symengine/tests/test_functions.py

```diff
@@ -120,7 +123,7 @@ def test_FunctionWrapper():
     assert e + sympy.loggamma(x) == x + 2*sympy.loggamma(x)
 
     f = e.subs({x : 10})
-    assert f == 10 + log(362880)
+    assert f == 10 + loggamma(10)
```

Comment:
Since `loggamma` is a know function now, we need to use an unknown function in line 120.